src/compute/app_runner.rs

```diff
@@ -333,8 +338,11 @@ mod tests {
             || {
                 let result = start();
                 assert!(
-                    result == 0 || result == 1 || result == 2 || result == 3,
-                    "start() should return a valid exit code"
+                    result == ExitMode::Success
```

Comment:
The way it is written, can we really obtain any value or is it always the same one due to specific parameters provided as environment variables ? The test seems to always return `UnreportedFailure`